tst/EnergyPlus/unit/CondenserLoopTowers.unit.cc

```diff
@@ -4880,4 +4880,547 @@ TEST_F(EnergyPlusFixture, CondenserLoopTowers_VSCoolingTower_OutputReport)
               OutputReportPredefined::RetrievePreDefTableEntry(*state, orp.pdchCTFCCondLoopBranchName, TowerName));
 }
 
+TEST_F(EnergyPlusFixture, CondenserLoopTowers_VSMerkelCoolingTower_OutputReport)
```

Comment:
Add a unit test for #10902 showing that it CoolingTower:VariableSpeed:Merkel is not reported to Equipment Summary - Cooling Towers and Fluid Coolers